plugins/gui/src/python/python_code_editor.cpp

```diff
@@ -19,6 +19,8 @@ namespace hal
         connect(redo_shortcut, &QShortcut::activated, this, &PythonCodeEditor::handle_redo_requested);
 
         m_base_file_modified = false;
+
+        toggle_minimap();
```

Comment:
Please do not do this. We have a setting for this. Set `[python] minimap=false` in `plugins/gui/defaults/guidefaults.ini` to switch from "default open" to "default closed". Your implementation hacks this in by inverting this setting and will confuse users.

Reply:
Didn't even know we had this. Sorry and thanks.